### ISSUE TITLE
Structure pattern: popovers should be closed when ESC is pressed

### ISSUE THREAD

**Issue by erral** [plone/mockup#968]
In the structure pattern if you are browsing with the keyboard using the TAB key, when a popover is opened after clicking an action button, it's not possible to close the popover, for instance using the ESC key.

@ionlizarazu
